Tests/aweXpect.Reflection.Tests/ThatAssembly.Has.AttributeTests.cs

```diff
@@ -0,0 +1,87 @@
+using System;
+using System.Reflection;
+using Xunit.Sdk;
+
+namespace aweXpect.Reflection.Tests;
+
+public sealed partial class ThatAssembly
```

Comment:
For all new tests the negated test cases using `DoesNotComplyWith` (see [ThatFields.ArePublic.NegatedTests](https://github.com/aweXpect/aweXpect.Reflection/blob/7ca54066e02e76194e0e78a2d1dc751c13b8009e/Tests/aweXpect.Reflection.Tests/ThatFields.ArePublic.Tests.cs#L45) for an example) are missing.
These tests should be added.